style.css

```diff
@@ -1,3 +1,9 @@
+body {
+  background-image: url("./assets/header-mobile-view.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
```

Comment:
- [ ] [1] Part of the project requirements is to follow a mobile-first approach for responsiveness. Upon reviewing your design on my device, it appears that it is not responsive in the mobile view. Therefore, I recommend adjusting the border and margin accordingly.